cluster/examples/kubernetes/ceph/object-test.yaml

```diff
@@ -22,3 +22,8 @@ spec:
     port: 80
     securePort:
     instances: 1
+  healthCheck:
```

Comment:
You still need to revert changes to this file.

Reply:
this time did (last time forgot to save the file)